lib/css-converter.js

```diff
@@ -195,126 +141,188 @@ function parseAttr (cssAttr) {
  * @property {?string} value The value of the pseudo selector
  */
 
-/**
- * Convert a CSS pseudo class to a UiSelector
- *
- * @param {CssPseudo} cssPseudo CSS Pseudo class
- * @returns {string} Pseudo selector parsed as UiSelector
- */
-function parsePseudo (cssPseudo) {
-  if (cssPseudo.valueType && cssPseudo.valueType !== 'string') {
-    throw new Error(`'${cssPseudo.name}=${cssPseudo.value}'. ` +
-      `Unsupported css pseudo class value type: '${cssPseudo.valueType}'. Only 'string' type or empty is supported.`);
-  }
-
-  const pseudoName = assertGetAttrName(cssPseudo);
+class CssConverter {
 
-  if (BOOLEAN_ATTRS.includes(pseudoName)) {
-    return `.${toSnakeCase(pseudoName)}(${assertGetBool(cssPseudo)})`;
+  constructor (selector, pkg) {
+    this.selector = selector;
+    this.pkg = pkg;
   }
 
-  if (NUMERIC_ATTRS.includes(pseudoName)) {
-    return `.${pseudoName}(${cssPseudo.value})`;
+  /**
+   * Add android:id/ to beginning of string if it's not there already
```

Comment:
Should we address `this.pkg` case as well?

Reply:
updated